client/app/telecom/telephony/billingAccount/orderAlias/nonGeographical/telecom-telephony-billing-account-orderAlias-nonGeographical.controller.js

```diff
@@ -129,6 +129,17 @@ angular.module('managerApp').controller('TelecomTelephonyAliasOrderNonGeographic
     }
   };
 
+  /**
+   * Get the periodicity of the billing
+   * @returns {String}
+   */
+  this.getPeriod = function () {
+    if (this.offer === 'didsOnly') {
```

Comment:
Duplicated code, this could be in a `Service` :)

Reply:
Why? I test the value of the radio button.